zds/member/api/serializers.py

```diff
@@ -17,10 +17,11 @@ class UserListSerializer(serializers.ModelSerializer):
     serializers.
     """
     avatar_url = serializers.CharField(source='profile.get_avatar_url')
+    link = serializers.CharField(source='get_absolute_url')
```

Comment:
Ok donc si c'est la ressources html, c'est pas bien, on est d'accord.

Le 28/03/2016 11:32, Gérard Paligot a écrit :
>
> In zds/member/api/serializers.py 
> <https://github.com/zestedesavoir/zds-site/pull/3265#discussion_r57557624>:
>
> > @@ -17,10 +17,11 @@ class UserListSerializer(serializers.ModelSerializer):
> >      serializers.
> >      """
> >      avatar_url = serializers.CharField(source='profile.get_avatar_url')
> > +    link = serializers.CharField(source='get_absolute_url')
>
> C'est la ressource html et non pas la ressource api. Et ce qui me 
> gêne, c'est de spécifier cette information dans la liste des 
> utilisateurs, un objet normalement plus léger.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub 
> <https://github.com/zestedesavoir/zds-site/pull/3265/files/f896364cc5fa324a343a91e6861e7050ce7e3ae6#r57557624>
>